conf/common/test/ucc_test_alltoallv.toml

```diff
@@ -0,0 +1,9 @@
+name = "ucc_test_alltoallv"
+description = "alltoallv"
+test_template_name = "UCCTest"
+
+extra_container_mounts = ["path_to_host_matrices:path_to_container_matrices_as_used_in_gen"]
```

Comment:
We can't use such values in configs under `common/` folder: they are supposed to work out of the box without any modifications.

Does this benchmark require some input before running it? Do we have an input example as part of container?

Reply:
I see. Let's try something like `extra_container_mounts = ["$PWD/conf/common:/workspace"]` to ensure it will work for everyone. If not possible, we have two options:
1. Put configs into another folder (not `common/`) and add description how to run it.
2. Not having an example at all (not good).